hokusai/production.yml

```diff
@@ -29,18 +28,35 @@ spec:
         ports:
         - containerPort: 3000
         resources:
+          requests:
+            cpu: "1"
```

Comment:
Just for some more context, this means "allocate 1 core to a Metaphysics pod". See https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

Compared to the running pods on production, it seems each metaphysics pod is using ~0.6 CPU so this gives us a lot of overhead.

![screen shot 2018-03-28 at 5 03 51 pm](https://user-images.githubusercontent.com/997613/38037921-24a8df78-32aa-11e8-8221-a07191397f0f.png)

However, we could set this closer to what we actually see as "normal" operation - i.e. set it to `0.6` and scale horizontally to spread load across more replicas.